### ISSUE TITLE
Review 

### ISSUE THREAD

**Issue by brendon-lind** [codalab/chagrade#146]
**Move Buttons to right-side of edit/create form on Homework Definition #125**
- [ ] Go to define homework page and look to see if the [ add question, add criterion, submit, cancel ] buttons are properly oriented on the right rather than the left.
- [ ] Go to the students enroll page and view the position of the buttons. They all are now on the top of their respective tables.
- [ ] Go to create a new class. The whole form is now styled in a much more modern design and the submit button is also moved to the right.

**Detailed grade #115**
- [ ] Go create a homework with 3 different Criteria with ranging scores. Go submit a submission to that assignment. Grade that assignment inputing different values for each criterion. Go to student view and view the homework submission. Should see a detailed grade to the left of the comments box .

**Order fixing on custom questions #131**
- [ ] Go create a homework with 3 custom text questions named Q1 / Q2 / Q3 in order. Submit to that homework with the answers A1/ A2 / A3. Go to view submission and they should all be numerical order.

**URL turned automatically to links #119** 
- [ ] Go create a homework with a couple URL type custom questions.First try to submit something that is not a valid URL like ( google.com ) and make sure you get an error message.Now submit homework with  URLs copied from the url bar on some web page like ( https://www.google.com/ ) and that should succeed. Grade that submission and notice they are now links. Go to student view and view that submission and see they are now a new tab clickable link as well so the student can test their own links.

**No way to delete homework #111**
- [ ] Go create a homework. Go to the page where you view all the homework and there should now be a red delete X. Click it and it will ask you if you're sure click ok and you should be redirected back to that page with a message in the top right of the page saying you deleted homework successfully. Assignment should not be listed anymore.

**Grading UI improvements #106**
- [ ] create homework with 5 or 6 custom questions to increase the width of the table to the point at which it must scroll horizontally.
- [ ] Submit to this homework as a student.
- [ ] Navigate to homework answers grading table for this homework and scroll to the right. Observe sticky `student name` column.

**No date in homework submission's information #84**
- [ ] Go grade a submission and now the submission info at the top of the page has the assignment name next to it and a few rows down there is a submission date displayed.

**Backup script #128**
This has to be tested in console.
- [ ] `./bin/pg_dump.sh` works local, saves file to minio private storage `/backups` dir
- [ ] (if we switch branch on prod to this and run the script) On prod storage backups are being saved from cron job
- [ ] Few commands added: 
 * `./manage.py list_backups <search>`
 * `./manage.py download_backup <dump name from above search>`
 * `./bin/pg_restore.sh <dump name from download>`

**Empty columns "Last modified" and "Avg. Grade" in "Define Homework" #100**
- [ ] Create or have homework with submissions already graded.
- [ ] Go to the define Homework page to see if it populates a date and an Average grade for all most recent submissions.

**Comment by gibsonbailey** [comment]
Move grading table legend to top. ✅

**Comment by gibsonbailey** [comment]
Order questions by question text ✅

**Comment by gibsonbailey** [comment]
Open jupyter notebook github url submissions with nbviewer. ✅

**Comment by gibsonbailey** [comment]
Non-JN homework does not format to 1 decimal place on student homework overview. ✅ Denominators formatted as integers on student homework overview.

**Comment by gibsonbailey** [comment]
Make definition and class description unlimited length (textarea). ✅

**Comment by gibsonbailey** [comment]
Change question mark button link to wiki to `Help`

**Comment by gibsonbailey** [comment]
Replace blue `submission detail` button with `Submission File` button on instructor. When it's a JN submission, label the button `Jupyter Notebook`.

**Comment by gibsonbailey** [comment]
`Link to github repo of submission` change this to github repo.

**Comment by gibsonbailey** [comment]
Submission detail url question validation prompt should include proper format example (include schema in url)
